pkg/migrations/migration_version.go

```diff
@@ -65,7 +66,7 @@ func SetCurrent(dbPath string) {
 		newVersion := &MigrationVersion{
 			dbPath:      dbPath,
 			MainVersion: version.GetMainVersion(),
-			SeqNum:      LastRocksDBVersionSeqNum(), // Most recent possible RocksDB version
+			SeqNum:      mathutil.MinInt(LastRocksDBVersionSeqNum(), CurrentDBVersionSeqNum()),
```

Comment:
The math isn't necessary since you do it in the call to `CurrentDBVersionSeqNum()`.

Reply:
I thought so but ...
The minimum in CurrentDBVersionSeqNum returns the correct sequence number with Postgres knob on/off. But this one is used in case of Postgres on. If we are running on Postgres and we are making the last mark to RocksDB to its maximum possible value.

Reply:
yeah I'm following.  IMO it is clearer to directly set it to `LastRocksDBVersionSeqNum`, but that is just my opinion.